cpp/src/hash/hash_allocator.cuh

```diff
@@ -26,6 +26,8 @@
 #include <rmm/mr/device/managed_memory_resource.hpp>
 #include <rmm/mr/device/per_device_resource.hpp>
 
+// TODO: Should all of this be namespaced in cudf?

```

Comment:
I was unable to locate how this `managed_allocator` code is used, if it is used at all. I searched `org:rapidsai` and `NVIDIA/spark-rapids` with the GitHub Code Search tool and found zero invocations of this. We might be able to remove it? I haven't searched for any of the other classes in this file, just `managed_allocator`.

Reply:
We are exclusively using `default_allocator` in all cuco use cases across libcudf.

Reply:
I'm going to resolve this discussion and move it to a separate PR.